### PR TITLE
[Enhancement] Add parquet int96(timestamp) ut

### PR DESCRIPTION
**Pull request by Smith-Cruise**
UT's parquet file is generated by Pyarrow. It has three columns with different timezone, actually, Pyarrow will write it all as UTC time.

Now our behavior is the same as Trino/Spark, but different from Hive. Hive is read int96 as utc time. Trino/Spark/SR will plus time zone offset for int96(timestamp) type.

## What type of PR is this:
- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [x] UT
- [ ] Doc
- [ ] Tool

## Checklist:
- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [x] I have checked the version labels which the pr will be auto-backported to the target branch
  - [x] 3.1
  - [x] 3.0
  - [ ] 2.5
  - [ ] 2.4

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Review by Youngwb — DISMISSED** [review]

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

File: be/test/formats/parquet/column_converter_test.cpp [hunk 1]
```diff
@@ -63,6 +63,9 @@ class ColumnConverterTest : public testing::Test {
     static void check_chunk_values(std::shared_ptr<Chunk>& chunk, const std::string& expected_value) {
         chunk->check_or_die();
         size_t mid = chunk->num_rows() / 2;
+        if (mid == 0) {
+            mid = 1;
```
Comment (dirtysalt) [inline be/test/formats/parquet/column_converter_test.cpp:67]: why this conversion?

Comment (Smith-Cruise) [reply be/test/formats/parquet/column_converter_test.cpp:67]: just trick code

**Review by dirtysalt — APPROVED** [review]

**Review by Youngwb — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
@Mergifyio backport branch-3.1

**Comment by wanpengfei-git** [comment]
@Mergifyio backport branch-3.0

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Pull request merged**